packages/router/src/__tests__/router.test.tsx

```diff
@@ -31,29 +31,23 @@ function createDummyAuthContextValues(partial: Partial<AuthContextInterface>) {
   return { ...authContextValues, ...partial }
 }
 
+const useAuth = (isAuthenticated = false, loading = false) => () =>
```

Comment:
Ok to call this `mockedUseAuth`? Not really a big deal, kust something I try to do when it's worth noting that something is intentionally being mocked

Reply:
Sure, I'll change it